presqt/api_v1/tests/views/target/test_target.py

```diff
@@ -54,7 +54,7 @@ def test_get_success(self):
         # Making this variable so we can append detail to
         # the list in the test case.
         json_data_keys = list(json_data[0].keys())
-        json_data_keys.append('detail')
+        json_data_keys.append('links')
 
         url = reverse('target', kwargs={'target_name': target_name})
         response = self.client.get(url)
```

Comment:
Maybe in the tests we should have a test of what is inside of links? Testing that the correct links are shown for a given target?